packages/dds/tree/api-report/tree.api.md

```diff
@@ -505,6 +517,8 @@ export interface FieldMapObject<TChild> {
 
 // @public
 export interface FieldProps {
+    // (undocumented)
+    readonly defaultProvider?: (context: DefaultProviderContext) => DefaultProviderValue;
```

Comment:
Since there is no way for any user of this package to produce a DefaultProviderValue, we should probably simplify this to simply:
```suggestion
    readonly defaultProvider?: DefaultProvider;
```

Where DefaultProvider is an erased type.

Reply:
I don't think we want to stabilize an API for providing defaults for fields in general yet. Doing so will be a bit more complex that just letting them provide insertable content somehow. When we do that (and we don't need it urgently) we will want it to be done in a type safe way. We will need to update ScheamFactory.required to allow it to take in a strongly typed default value provider. I don't think we want to do that as part of this change though.

Reply:
Updated to use erasedType `DefaultProvider`